pages/[lng]/register/index.tsx

```diff
@@ -116,6 +116,7 @@ const RegisterPage: FC<any> = ({
   const i18n: any = useI18n()
   const router: any = useRouter()
   const query = router?.query || {}
+  const recaptchaRef = useRef<any>();
```

Comment:
```suggestion
  const recaptchaRef = useRef<any>()
```

titik koma dihilangken supaya seragam